pkg/controller/federatedtypeconfig/controller.go

```diff
@@ -104,7 +105,7 @@ func ProvideController(arguments args.InjectArgs, fedNamespace, clusterNamespace
 }
 
 func (c *FederatedTypeConfigController) Reconcile(k types.ReconcileKey) error {
-	log.Printf("Running reconcile FederatedTypeConfig for %s\n", k.Name)
+	glog.Infof("Reconciling FederatedTypeConfig %v/%v", k.Namespace, k.Name)
```

Comment:
@pmorie I think we also need to unify the log level in all controllers, you may see that we are using different log levels of `glog.V(4).Infof` at https://github.com/kubernetes-sigs/federation-v2/blob/master/pkg/controller/schedulingpreference/controller.go#L268

Reply:
@gyliu513 how about opening an issue for this to handle it across.

Reply:
Thanks @irfanurrehman , I'm +1 for this, and I also like to use `glog.Infof()` as @pmorie is doing here.

@marun do you think it is reasonable to create a PR to unify the log level for all controllers for `Reconcile`?

Reply:
I'm not sure of the value of synchronizing log level everywhere without clear criteria for doing so.  I think @irfanurrehman's suggestion of filing an issue to discuss is a good next step.

Reply:
Done, I opened an issue at https://github.com/kubernetes-sigs/federation-v2/issues/251 for discussion.